plugin.video.mlbtv/resources/language/English/strings.po

```diff
@@ -0,0 +1,101 @@
+# Kodi Media Center language file
```

Comment:
can you please also adjust the language folder layout? See: http://kodi.wiki/view/language_support ; https://github.com/xbmc/generator-kodi-addon/tree/master/generators/app/templates/resources/language